docs/changehistory/NextVersion.md

```diff
@@ -1,5 +1,37 @@
 ---
 publish: false
 ---
+
 # NextVersion
```

Comment:
@calebmshafer is resolving the merge issue that caused these to show up as diffs for your PR.

Reply:
Opened #695 to fix the issue. That's my mistake when I did the merge for 2.11.

Reply:
resolved the merge conflicts, sorry this took so long.